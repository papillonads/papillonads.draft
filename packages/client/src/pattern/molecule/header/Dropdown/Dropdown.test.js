import defaultRender from './Dropdown.render'

describe('<Dropdown />', () => {
  describe('Snaphot', () => {
    test('must match defaultRender', () => {
      expect(global.renderToJSON(defaultRender)).toMatchSnapshot()
    })
  })
})
