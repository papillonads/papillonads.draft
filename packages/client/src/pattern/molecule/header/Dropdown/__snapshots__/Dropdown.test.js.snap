// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Dropdown /> Snaphot must match defaultRender 1`] = `
<details
  className="details-reset dropdown"
>
  <summary
    aria-haspopup="true"
    className="dropdownSummary"
  >
    <div
      className="dropdownBadge"
    >
      <div
        className="dropdownSvgIcon1"
      >
        <svg
          fill="grey"
          height={20}
          viewBox="0 0 1024 1024"
          width={20}
        >
          <path
            d="M576 706.612v-52.78c70.498-39.728 128-138.772 128-237.832 0-159.058 0-288-192-288s-192 128.942-192 288c0 99.060 57.502 198.104 128 237.832v52.78c-217.102 17.748-384 124.42-384 253.388h896c0-128.968-166.898-235.64-384-253.388z"
          />
        </svg>
      </div>
      <div
        className="dropdownLabel"
      >
        t
      </div>
      <div
        className="dropdownSvgIcon2"
      >
        <svg
          fill="grey"
          height={15}
          viewBox="0 0 1024 1024"
          width={15}
        >
          <path
            d="M287.986 576c8.908 0 17.77 3.698 24.096 10.928l199.918 228.478 199.918-228.478c11.636-13.3 31.856-14.65 45.154-3.010 13.3 11.638 14.648 31.854 3.010 45.154l-224 256c-6.076 6.944-14.854 10.928-24.082 10.928s-18.006-3.984-24.082-10.928l-224-256c-11.638-13.3-10.292-33.516 3.010-45.154 6.070-5.312 13.582-7.918 21.058-7.918z"
          />
        </svg>
      </div>
    </div>
  </summary>
  <ul
    className="dropdown-menu dropdownList"
  >
    <li>
      <div
        className="dropdown-item dropdownListItem"
      >
        item 1
      </div>
    </li>
    <li>
      <div
        className="dropdown-item dropdownListItem"
      >
        item 2
      </div>
    </li>
    <li>
      <div
        className="dropdown-item dropdownListItem"
      >
        item 3
      </div>
    </li>
  </ul>
</details>
`;
